src/app/snapshot/snapshots-page/snapshot-list-item/snapshot-item.component.ts

```diff
@@ -29,13 +29,12 @@ export class SnapshotItemComponent {
       SnapshotStates.Error,
     ]
       .filter(state => this.item.state === state)
-      .map(
-        state =>
-          state === SnapshotStates.BackingUp
-            ? 'backing-up'
-            : state === SnapshotStates.BackedUp
-              ? 'backed-up'
-              : state.toLowerCase(),
+      .map(state =>
```

Comment:
remove these changes

Reply:
This changes caused your pre-commit hook